docs/menu.yml

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+catalog:
+  - name: Welcome
+    path: /readme
+  - name: Concepts and Designs
+    path: /en/concepts-and-designs/readme
```

Comment:
As we are going to host the docs in our website, I prefer to keep https://skywalking.apache.org/blog/e2e-design/ up to date and link `Concepts and Designs` to there, (though there are some unfinished features in it, we can mark them as `TODO`). WDYT @mrproliu @fgksgf @wu-sheng ?

Reply:
Make that doc in the official doc as further plan guidance for potential contributors, it works for me.
But please update all content in the doc up to date, and after we do that, we don't need a pointer to that blog. The blog becomes the history context. 
Does this work?

Reply:
Don't put the path when you have catalog. It makes the page(browser) shaking.

Reply:
I will keep the documentations.

Reply:
What do you mean `keep the documentations`?

Reply:
> What do you mean `keep the documentations`?

That's mean don't need to use that blog, only use the document to descript the concepts and designs.